tests/unit/s2n_resume_test.c

```diff
@@ -23,6 +23,14 @@
 
 #define S2N_TLS13_STATE_SIZE_WITHOUT_SECRET S2N_MAX_STATE_SIZE_IN_BYTES - S2N_TLS_SECRET_LEN
 
+static int s2n_test_session_ticket_callback(struct s2n_connection *conn,
+                                         uint8_t *session_id_data, size_t session_id_len,
+                                         uint8_t *session_data, size_t session_len,
+                                         uint32_t session_lifetime)
```

Comment:
Nit: Your indentation is wonky here. Either they should all line up, or they should just wrap.